src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -146,14 +146,19 @@ namespace ts.codefix {
         return false;
     }
 
-    function tryPrefixDeclaration(changes: textChanges.ChangeTracker, errorCode: number, sourceFile: SourceFile, token: Node): void {
+    function tryPrefixDeclaration(changes: textChanges.ChangeTracker, errorCode: number, sourceFile: SourceFile, checker: TypeChecker, token: Node): void {
         // Don't offer to prefix a property.
         if (errorCode === Diagnostics.Property_0_is_declared_but_its_value_is_never_read.code) return;
         if (token.kind === SyntaxKind.InferKeyword) {
             token = cast(token.parent, isInferTypeNode).typeParameter.name;
         }
         if (isIdentifier(token) && canPrefix(token)) {
             changes.replaceNode(sourceFile, token, createIdentifier(`_${token.text}`));
+            FindAllReferences.Core.eachSymbolReferenceInFile(token, checker, sourceFile, (ref: Node) => {
+                if (isIdentifier(ref)) {
+                    changes.replaceNode(sourceFile, ref, createIdentifier(`_${ref.text}`));
+                }
+            });

```

Comment:
Hmm, using `FindAllReferences` is probably a lot heavier of a tool than we need here. There’s a `getJSDocParameterTags` utility function that would probably do the trick. Could you try that instead?

Reply:
Hi, recode as you suggested.